bin/systemds

```diff
@@ -396,7 +396,7 @@ fi
 JARNAME=$(basename "$SYSTEMDS_JAR_FILE")
 
 # relative path to jar file
-SYSTEMDS_JAR_FILE=$(realpath --relative-to=. "$(dirname "$SYSTEMDS_JAR_FILE")")${DIR_SEP}${JARNAME}
+SYSTEMDS_JAR_FILE=$(grealpath --relative-to=. "$(dirname "$SYSTEMDS_JAR_FILE")")${DIR_SEP}${JARNAME}
```

Comment:
does this fix it for mac ?

Reply:
Yes, it does. Should I change it back in my next commit?